### DISCUSSION TITLE
Github Actions docs say that workflows won't run in forks by default, but it seems this is not the case?

### DISCUSSION THREAD

**Discussion by bzm3r** [community/community#31889] [Actions]
The relevant docs: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflows-in-forked-repositories

What happened: I synced a fork up with main repository, after creating some CI for the main repository. After I synced the fork, I checked the Actions tab, and actions were running :(

It seems that one must use `if: github.repository_owner = "Octocat"` on jobs in the workflow? Am I understanding correctly?

**Comment by flobernd** [comment]
I think the documentation is misleading here. It refers to the fact that workflows do not run automatically if **you** fork a **foreign** repository. 

The case you are looking at is about creating a PR from a forked repo. There are options in the repository settings to control that behavior:

![](https://nextcloud.flobernd.de/s/pmMpEaddiqTFGYC/preview)
